src/device/GPU_Device.cc

```diff
@@ -9,6 +9,7 @@
 //---------------------------------------------------------------------------//
 
 #include "GPU_Device.hh"
+#include <iostream>
```

Comment:
Is this required?  I thought we prevented this class from writing directly to stdin/stderr?